pkg/ottl/ottlfuncs/README.md

```diff
@@ -626,6 +627,33 @@ Examples:
 
 - `Int("2.0")`
 
+### Hex
+
+`Hex(value)`
+
+The `Hex` converter converts the `value` to its hexadecimal representation.
+
+The returned type is string.
+
+The input `value` types:
+
+- float64. Fraction is discharged (truncation towards zero).
+- string. Each byte of string is converted to hex.
```

Comment:
I think this handling makes sense, but in a way it's also an exception to our handling since with the exception of byte slices, other types are converted to ints before being converted to hex. We should at least call out that integer values represented as strings (e.g. `"123456"`) should be converted before being passed to this, like `Hex(Int(attributes["my_int_string"]))`.

Reply:
agree, adapted it!